### PR TITLE
Export and Auto Review changes

### PR DESCRIPTION
**Pull request by gtpeddi94**
Updated logic for exceptions to use get_submission_results() to get reviewer ID.

Created functions for reject_max_length_threshold and reject_min_length_threshold and replicated changes in reviewer and indico_wrapper accordingly.

Created field_config. yaml files for Controllership, GOS and MRI

### CODE REVIEW

File: scripts/generate_export.py
Comment (fitzworkhub) [inline scripts/generate_export.py:36]: No reason to change the default argument here. This behavior is controlled by the `post_review` variable ~ line 314.

File: scripts/generate_export.py
Comment (fitzworkhub) [inline scripts/generate_export.py:13]: Any particular reason for changing the USAGE string? This variable is only used when the program isn't called correctly.

File: scripts/generate_export.py
Comment (fitzworkhub) [inline scripts/generate_export.py:8]: Why are we commenting this out? Reviewer is necessary for post processing.  Particularly in the YArdi use case. Check out line 355

File: solutions_toolkit/auto_review/auto_review_functions.py
Comment (fitzworkhub) [inline solutions_toolkit/auto_review/auto_review_functions.py:1]: We need to start writing unittests for these functions. I'll post an example one before our next meeting. But it's important that we know these auto review functions are working as expected since they are cross use case.

File: scripts/generate_export.py
Comment (fitzworkhub) [inline scripts/generate_export.py:494]: Are you sure you want to keep print statements in here?

**Review by fitzworkhub — CHANGES_REQUESTED** [review]
Good stuff here! A few things need to be fixed before approval though.  We also need to start thinking about unittesting going forward.

File: scripts/generate_export.py
Comment (gtpeddi94) [reply scripts/generate_export.py:36]: I may have accidentally changed it. Please ignore this.

File: scripts/generate_export.py
Comment (gtpeddi94) [reply scripts/generate_export.py:13]: I was trying something out and forgot to change it back. Please ignore this as well.

File: scripts/generate_export.py
Comment (gtpeddi94) [reply scripts/generate_export.py:8]: Yes, I understand. I commented it out because it was throwing an error on my end for some reason. It could not find Reviewer so for the time being for my test run, I commented it out. Please ignore this too! Gos! Sorry. I need to do a better job when pushing changes next time.

File: solutions_toolkit/auto_review/auto_review_functions.py
Comment (gtpeddi94) [reply solutions_toolkit/auto_review/auto_review_functions.py:1]: I tested it for MRI and it is working as expected

File: scripts/generate_export.py
Comment (gtpeddi94) [reply scripts/generate_export.py:494]: Nope. They can be removed. Can you disapprove certain changes like these for which you commented? Or do I need to push changes again and create a PR?

Comment (fitzworkhub) [reply scripts/generate_export.py:494]: Next steps is making changes on your branch locally. Then commit and push the changes and this PR will get updated with your new code that I can review.

Comment (fitzworkhub) [reply scripts/generate_export.py:494]: Sorry, I should have had the steps more formalized.  If you make the changes to this current branch the PR will update so you don't need to make a new (or a new branch).

from the solutions-toolkit repo on your local
1) `git checkout Gnana_changes_3_5_21` (this branch)
2) make requested edits above
3) git add + commit
4) git push 

This current PR will reflect the new changes.

**Comment by gtpeddi94** [comment]
I actually had to delete the existing repo on my machine because when I staged changes it showed nothing to commit and the edits didn’t get saved. So I cloned and pulled the repo again, made the changes again, and pushed those changes to a new branch. Sorry for the confusion! Could you approve these changes? Barring the edits you requested to make the rest of the modifications are the same as the previous push.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fitzworkhub** [comment]
To be clear:
- you deleted the repo because you were having trouble committing changes?
- when you cloned the repo, rather than checking out this current branch you checked out a new branch?
- you made changes to the new branch and made another pull request?

**Comment by gtpeddi94** [comment]
Yes, that’s right. A fresh pull request from a new branch.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fitzworkhub** [comment]
Got it. For next time, check out the branch you made the original PR with.

**Comment by gtpeddi94** [comment]
Sure! Apologies for the inconvenience.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**
